database/postgres/ddl/repo.go

```diff
@@ -32,6 +32,7 @@ repos (
 	allow_tag     BOOLEAN,
 	allow_comment BOOLEAN,
 	pipeline_type TEXT,
+	previous_name VARCHAR(40),
```

Comment:
Are we sure we want to restrict the repo name to `40` characters?

I ask because I'm not sure what GitHub's limit is but I know it's a higher amount than that.

Here's a repo I created where it has `100` characters in the name:

https://github.com/jbrockopp/1234567890123456789012345678901234567890123456789012345678901234567890123456789012345678901234567890

Reply:
Just tested it out — 100 is the max